feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -100,6 +104,32 @@ internal class HomeViewModel @Inject constructor(
         }
     }
 
+    private fun handleTimerCancellationEvent(event: TimerCancellationEvent) {
```

Comment:
Can we move this timer cancellation logic to `TimerHandler` by merging `TimerCancellationEvent` with `TimerEvent`?
If not, please create a separate handler for it to keep the `HomeViewModel` small.

Reply:
Done